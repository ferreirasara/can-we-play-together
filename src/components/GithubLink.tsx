import { Link } from "@mui/material";

export default function GithubLink() {
  return (
    <Link underline="none" target="_blank" rel="noopener" href="https://github.com/ferreirasara">
      Sara Ferreira
    </Link>
  )
}