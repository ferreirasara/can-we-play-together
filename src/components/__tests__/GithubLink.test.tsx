import renderer from 'react-test-renderer';
import GithubLink from '../GithubLink';

it('renders correctly', () => {
  const filters = renderer
    .create(<GithubLink />)
    .toJSON();
  expect(filters).toMatchSnapshot();
});