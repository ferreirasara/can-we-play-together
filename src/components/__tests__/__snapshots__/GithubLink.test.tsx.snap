// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-1cmksfb-MuiTypography-root-MuiLink-root"
  href="https://github.com/ferreirasara"
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  rel="noopener"
  target="_blank"
>
  Sara Ferreira
</a>
`;
